.travis.yml

```diff
@@ -2,6 +2,11 @@ language: cpp
 sudo: required
 dist: trusty
 
+addons:
+  apt:
+    sources:
+    - sourceline: 'ppa:kzemek/boost'
```

Comment:
Maybe move this to the shell script:
```
if [[ $TRAVIS == "true" ]]; then
    sudo add-apt-repository ppa:kzemek/boost -y
fi
```
somewhere here
https://github.com/Microsoft/LightGBM/blob/d852716c21e52fda47d7eccee7e6b674c1e9e35d/.ci/setup.sh#L44
for not bothering other tasks, except gpu?